CMakeLists.txt

```diff
@@ -431,3 +431,6 @@ endif()
 
 include(cmake/AddInstallationScriptTest.cmake )
 add_installation_script_test(installation-scripts.sh src/tests/installation/)
+
+include(AddCompileTimeTest.cmake)
```

Comment:
Can we please add new cmake modules to the top level `cmake` subdirectory, with the other ones? It just helps keep the top level project more organized and less cluttered IMO, but I'm open to arguments that contradict this point of view.